assets/css/style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background: #f7f7f9;
+  font-family: 'poppins', sans-serif;
+}
+
+/* Header/nav */
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+  width: 375px;
+  color: #6070ff;
+  background: #fff;
+  padding: 24px;
+}
+
+nav h1 {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+}
+
+/* Hero section */
+
+.hero {
+  background: #fff;
+  height: 600px;
+  width: 375px;
+  top: 92px;
+  padding: 114px 24px;
+  border-bottom-left-radius: 120px;
+  background-image: url("../images/backg.svg") no-repeat;
```

Comment:
- [ ] Kindly remove the `no-repeat` from the `background-image` property because of this `mistake` your background-image is not visible. You can put it separately by using the *background-repeat: no-repeat;*  property. Please have look at below suggestions.

``` suggestion
  background-image: url("../images/backg.svg");
  background-repeat: no-repeat;
```

## OR

``` suggestion
  background: url("../images/backg.svg") no-repeat;
```

### Your Output 

![Screenshot from 2023-05-03 14-55-52](https://user-images.githubusercontent.com/66476329/235879617-93dc6d5c-c2d1-409a-b67c-d2b96f7cc75a.png)